frontend/src/components/StaticMetadata.vue

```diff
@@ -8,21 +8,42 @@
                 </v-card-subtitle> -->
       </v-card-item>
       <v-card-text>
+        <!-- Default Metadata -->
         <div v-for="metadataObject in defaultSwordMetadata()" :key="metadataObject.id">
           <v-divider />
-          <div>
-            <strong>{{ metadataObject.short_definition }}:</strong> {{ metadataObject.value }}
+          <div style="display: flex; align-items: center; justify-content: space-between;">
+            <div>
+              <strong>{{ metadataObject.displayKey }}:</strong> {{ metadataObject.value }}
+            </div>
+            <!-- Tooltip Icon -->
+            <v-tooltip location="top" max-width="200px" style="white-space: normal;">
+              <template #activator="{ props }">
+                <v-icon v-bind="props" :icon="mdiInformationOutline" color="primary" style="cursor: pointer; margin-left: 10px;">
+                </v-icon>
+              </template>
+              <div>{{ metadataObject.definition }}</div>
+            </v-tooltip> 
```

Comment:
it looks like you have a lot of your metadata updates from https://github.com/CUAHSI/SWOT-Data-Viewer/pull/95 included here in this PR... I assume this is either unintentional on your part, or some sort of bug in GH (I've seen diff bugs in GH before...)

Reply:
<img width="601" alt="image" src="https://github.com/user-attachments/assets/6a77e8f1-4505-4b76-9d03-e2223e722c7f">
yep looks like you continued off your `IG-metadata-cleanup` branch

Reply:
you probably want to rebase `IG-charts-legend` onto `develop`
You will have merge conflicts that might be difficult to resolve... LMK if you need help

Reply:
I would suggest that you do this rebase BEFORE addressing any of the other issues in this PR

Reply:
Actually I went ahead and rebased for you (hope that I got all the conflicts right! 🤞 )
consider using this branch instead: https://github.com/CUAHSI/SWOT-Data-Viewer/tree/IG-charts-legend-rebase

Reply:
Scratch that, the [rebased branch](https://github.com/CUAHSI/SWOT-Data-Viewer/tree/IG-charts-legend-rebase ) did not have the conflicts resolved correctly. So don't use it! 😄 

[IG-charts-legend-2](https://github.com/CUAHSI/SWOT-Data-Viewer/tree/IG-charts-legend-2) should be more accurate, so I would fast-forward merge your branch to that.